alyx/behavior/models.py

```diff
@@ -68,7 +68,7 @@ class EventSeries(BaseExperimentalData):
     """
     event_times = models.ForeignKey(Dataset, blank=True, null=True,
                                     related_name="event_series_event_times",
-                                    help_text="n*1 array of times in seconds")
+                                    help_text="n*1 array of times in seconds (universal timescale)")
     type_descriptions_id = models.ForeignKey(Dataset, blank=True, null=True)
     event_types_id = models.ForeignKey(Dataset, blank=True, null=True,
                                        related_name="event_series_event_descriptions",
```

Comment:
IntervalSeries

* rename interval_description to interval_types
* add type_descriptions